src/log/ModDaemon.cc

```diff
@@ -338,7 +338,7 @@ logfile_mod_daemon_flush(Logfile * lf)
 {
     l_daemon_t *ll = static_cast<l_daemon_t *>(lf->data);
     if (commUnsetNonBlocking(ll->wfd)) {
-        debugs(50, DBG_IMPORTANT, "Logfile Daemon: Couldn't set the pipe blocking for flush! You're now missing some log entries.");
+        debugs(50, DBG_IMPORTANT, "ERROR: Logfile Daemon: Could not set the pipe blocking for flush! You're now missing some log entries.");
```

Comment:
Since we are using formal language now:   s/You're/You are/   or even "There are" if possible.

Reply:
Good catch. It is too risky for the dumb script to change "You" to "There", but it can expand contractions. Done in 892d596.